server/parsers/ppl_parser.ts

```diff
@@ -31,7 +31,7 @@ export const PPLParsers: MessageParser = {
 
     if (!ppls.length) return [];
 
-    const statsPPLs = ppls.filter((ppl) => /\|\s*stats\s+[^|]+\sby\s/i.test(ppl));
+    const statsPPLs = ppls;
     if (!statsPPLs.length) {
       return [];
     }
```

Comment:
since we don't need ppl based charts, maybe just remove these 
```suggestion
```
and change `statsPPLs` here to `ppls`? https://github.com/opensearch-project/dashboards-observability/blob/2e2e912e741756e233602545092c64197c5286c0/server/parsers/ppl_parser.ts#L39

Reply:
Thanks, done for that.